returnn/datasets/distrib_files.py

```diff
@@ -140,9 +147,12 @@ def __init__(
         :param get_sub_epoch_dataset: callable which returns a dataset dict for a given subset of files
         :param preload_next_n_sub_epochs: how many sub epoch datasets to preload
         :param buffer_size: buffer size for each worker, amount of seqs to prefetch
+        :param shard: set to true to shard the data across worker processes
```

Comment:
I wonder if we maybe should choose a bit more detailed name, like `distributed_sharding` or so? Or is sharding really only relevant for distributed training?

Reply:
Hmm, I'm not sure either. I like that it's concise and short, but I also feel like I lack background knowledge to answer whether sharding is relevant in other cases. We could rename it to `shard_files` though to be clearer on the behavior.

Reply:
But it's also specific to distributed training. I'm not sure if there are maybe other applications of sharding.

Reply:
So `distrib_shard_files`?